schema-extended/Common.json

```diff
@@ -0,0 +1,308 @@
+        {
+          "type": "object",
+          "properties": {
+            "abstract": {
+              "type": "string",
+              "minLength": 1
+            },
+            "language": {
+              "const": "en"
+            }
+          },
+          "additionalProperties": false
+        }
+      ],
+      "additionalItems": {
+        "type": "object",
+        "properties": {
+          "abstract": {
+            "type": "string",
+            "minLength": 1
+          },
+          "language": {
+            "allOf": [
+              {
+                "$ref": "LanguageCode.json"
+              },
+              {
+                "not": {
+                  "enum": [
+                    "en"
+                  ]
+                }
+              }
+            ]
+          }
+        },
+        "additionalProperties": false
+      }
+    },
+    "arxiv_status": {
+      "title": "arXiv posting status",
+      "description": "The status of this document version on arXiv.",
+      "type": "string",
+      "enum": [
+        "posted",
+        "withdrawn"
+      ],
+      "minLength": 1
+    },
+    "author_id": {
+      "description": "arXiv author identifier, if available.",
+      "type": "string",
+      "pattern": "^[a-z]+_[a-z]{1}_[0-9]+$"
+    },
+    "comment": {
+      "title": "Comment",
+      "type": "string",
+      "minLength": 1
+    },
+    "conference_info": {
+      "title": "Conference information",
+      "type": "object",
+      "properties": {
+        "conference_titles": {
+          "title": "List of conference titles",
+          "type": "array",
+          "items": {
+            "type": "string",
+            "minLength": 1
+          },
+          "minItems": 1,
+          "uniqueItems": true
+        },
+        "conference_pid": {
+          "title": "Conference Persistent Identifier",
+          "$comment": "placeholder to support: https://www.crossref.org/working-groups/conferences-projects/",
+          "description": "The persistent identifier for the conference.",
+          "type": "string",
+          "minLength": 1
+        }
+      },
+      "required": [
+        "conference_titles"
+      ]
+    },
+    "collaboration": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "The fullest representation of a collaborations's name",
+          "allOf": [
+            {
+              "type": "string",
+              "minLength": 1
+            },
+            {
+              "not": {
+                "type": "string",
+                "pattern": "(?i)(collaboration)"
+              }
+            }
+          ]
+        }
+      },
+      "required": [
+        "name"
+      ],
+      "additionalProperties": false
+    },
+    "document_type": {
+      "title": "Document type",
+      "$comment": "TODO: enumeration needs further review, and will need to decide whether the items require their own definitions with more structure.",
```

Comment:
for discussion

Reply:
Steinn: language not consistent across categories. Impose article-type hegemony or support different types across different categories?

Reply:
Steinn: 
* accommodate different kinds of reviews, e.g. living review
* we don't officially support book reviews right now
* can entertain poster type if we can handle the volume in the future